src/ompl/geometric/planners/fmt/src/FMT.cpp

```diff
@@ -478,10 +581,12 @@ bool ompl::geometric::FMT::expandTreeFromNode(Motion **z)
                 x->setParent(yMin);
                 x->setCost(cMin);
                 x->setHeuristicCost(opt_->motionCostHeuristic(x->getState(), goalState_));
+                yMin->getChildren().push_back(x);
+
                 // Add x to Open
-                Open_new.push_back(x);
-                // Remove x from Unvisited
-                x->setSetType(Motion::SET_CLOSED);
+                Open_.insert(x);
```

Comment:
I see that in fact the footnote changed. Speed is negligible because `Open_new` is relatively small.